src/cmd/singularity/cli/help.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"os"
+
+	"github.com/singularityware/singularity/src/docs"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	HelpCmd.Flags().SetInterspersed(false)
+	SingularityCmd.SetHelpCommand(HelpCmd)
+
+	SingularityCmd.AddCommand(HelpCmd)
+}
+
+// HelpCmd singularity help
+var HelpCmd = &cobra.Command{
+	DisableFlagsInUseLine: true,
+	Args:   cobra.ExactArgs(1),
```

Comment:
This is wrong isn't is? I mean... what if you do:
```
singularity help instance start
```
It'd only look at the help for `instance` and not `instance start`

Reply:
ummm I don't know how cobra can handle that ....

Reply:
ha!  found!